MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionPresenter.h

```diff
@@ -275,6 +275,13 @@ protected slots:
   void inputChecksBeforeFitting(const std::string &focusedRunNo,
                                 const std::string &ExpectedPeaks);
 
+  void updateFittingDirVec(std::string &bankDir, std::string &focusedFile,
+                           bool multi_run,
+                           std::vector<std::string> &fittingRunNoDirVec);
+
+  void enableMultiRun(std::string firstRun, std::string lastRun,
+                      std::vector<std::string> &fittingRunNoDirVec);
```

Comment:
All input parameters should be `const`, and the `string` and `vector` ones `const &`